### PR TITLE
OSDOCS-4718 vSphere DRS Anti-Affinity Rule configuration During UPI Install 2nd PR

### PR DESCRIPTION
**Pull request by GroceryBoyJr**
How to configure DRS anti-affinity rules during IPI installation

Status: *merge review requested.* 
This is a new PR of the docs in https://github.com/openshift/openshift-docs/pull/51584 which has already been merged. QE and SME approval are needed for 4.11 and 4.10.  *SME and QE LGTM have been given.*

Version(s):
4.11 (this PR)
Please cherrypick to 4.10 also.

Issue:
THIS PR: https://issues.redhat.com/browse/OSDOCS-4718
original: https://issues.redhat.com/browse/SPLAT-597

Links to docs previews:

( This is a "small size/M" because it's the same original module referenced in three places. )

https://57030--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_vsphere/installing-vsphere.html#anti-affinity-vsphere_installing-vsphere

https://57030--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_vsphere/installing-vsphere-network-customizations.html#anti-affinity-vsphere_installing-vsphere-network-customizations

https://57030--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_vsphere/installing-restricted-networks-vsphere.html#anti-affinity-vsphere_installing-restricted-networks-vsphere

SME Review: Joseph Callen @jcpowermac
QE Review: Shang Gao @sgaoshang

### CODE REVIEW

**Comment by GroceryBoyJr** [comment]
QE LGTM: https://github.com/openshift/openshift-docs/issues/54061#issuecomment-1453112125

**Comment by mjpytlak** [comment]
> If I need cherrypick to 4.10, should I also have labeled "branch/enterprise-4.10"?

Yes, please do.

Also - how did you create this PR. Did you use the commit id  from https://github.com/openshift/openshift-docs/pull/51584? If not, I believe QE should approving here..

**Comment by GroceryBoyJr** [comment]
> > If I need cherrypick to 4.10, should I also have labeled "branch/enterprise-4.10"?
> 
> Yes, please do.
> 
> Also - how did you create this PR. Did you use the commit id from #51584? If not, I believe QE should approving here..

I created this new PR from scratch per Kathryn's request. But I referred to the QE LGTM from the cherrypick, which is where I pulled the file from (to save the work).

**Comment by GroceryBoyJr** [comment]
@mjpytlak We're good with both SME and QE LGTM. I tried to use the original cherrypick, since I already had LGTM from those folks for it, but I was told to open a new PR and refer to the prior LGTM.

File: modules/vsphere-anti-affinity.adoc [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+//
+// * installing/installing_vsphere/installing-restricted-networks-vsphere.adoc
+// * installing/installing_vsphere/installing-vsphere-network-customizations.adoc
+// * installing/installing_vsphere/installing-vsphere.adoc
+
+:_content-type: PROCEDURE
+[id="anti-affinity-vsphere_{context}"]
+= Configuring vSphere DRS anti-affinity rules for control plane nodes
+
+vSphere Distributed Resource Scheduler (DRS) anti-affinity rules can be configured to support higher availability of {product-title} Control Plane nodes. Anti-affinity rules ensure that the vSphere Virtual Machines for the {product-title} Control Plane nodes are not scheduled to the same vSphere Host.
```
Comment (mjpytlak) [inline modules/vsphere-anti-affinity.adoc:11]:
```suggestion
vSphere Distributed Resource Scheduler (DRS) anti-affinity rules can be configured to support higher availability of {product-title} control plane nodes. Anti-affinity rules ensure that the vSphere Virtual Machines for the {product-title} control plane nodes are not scheduled to the same vSphere Host. 
```

**Review by mjpytlak — COMMENTED** [review]
One small item to address @GroceryBoyJr .

File: modules/vsphere-anti-affinity.adoc (hunk 1 above)
Comment (GroceryBoyJr) [reply modules/vsphere-anti-affinity.adoc:11]: Thank you! Much appreciated, @mjpytlak! Of course, the fix here replicates to all three instances. Lovin the modularity.

**Pull request merged**